.github/CONTRIBUTING.md

```diff
@@ -1,96 +1,66 @@
-# Contributing
-
+# Contributing Guide
 Please reference the information below when contributing to this repository.
 
-## Requirements
+## Welcome
+Welcome to the FlyByWire Simulations Discord bot repository. Thank you for your interest in contributing to the project. Full details and guidelines on how to ensure this project is managed are included below.
 
-* Write your commands professionally and clearly.
-* Proofread your work before submitting as ready for review.
-* Choose user-friendly command names.
-* Test your build locally before submitting as ready for review.
+## Helping others
+Please help other contributors to the project wherever you can, as people all start somewhere. If you require assistance or wish to provide assistance you can ask/answer questions on the #dev-support or #discord-bot channel on [discord](https://discord.gg/flybywire).
```

Comment:
```suggestion
Please help other contributors to the project wherever you can, as people all start somewhere. If you require assistance or wish to provide assistance you can ask/answer questions in the #discord-bot or #dev-support channel on [discord](https://discord.gg/flybywire).
```

Would prefer them to come to us in #discord-bot first for bot related issues. Putting it first makes it suggestive